modules/metering-store-data-in-shared-volumes.adoc

```diff
@@ -28,15 +32,112 @@ spec:
+
+<2> Set `storage.hive.sharedPVC.createPVC` to `true` and set the `storage.hive.sharedPVC.storageClass` to the name of a storage class with ReadWriteMany access mode. This will use dynamic volume provisioning to have a volume created automatically.
+--
+
+.  Create the following resource objects that are required to deploy an NFS server for metering. Use the `oc create -f <file-name>.yaml` command to create the object YAML files.
+..  Configure a `PersistentVolume` resource object. For example, `nfs_persistentvolume.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: nfs
+  labels:
+    role: nfs-server
+spec:
+  capacity:
+    storage: 5Gi
+  accessModes:
+  - ReadWriteMany
+  storageClassName: nfs-server <1>
+  nfs:
+    path: "/"
+    server: REPLACEME
+  persistentVolumeReclaimPolicy: Delete
+----
+<1> Must exactly match the `[kind: StorageClass].metadata.name` field value.
+
+..  Configure a `Pod` resource object with the `nfs-server` role. For example, `nfs_server.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nfs-server
+  labels:
+    role: nfs-server
+spec:
+  containers:
+    - name: nfs-server
+      image: <image_name> <1>
+      imagePullPolicy: IfNotPresent
+      ports:
+        - name: nfs
+          containerPort: 2049
+      securityContext:
+        privileged: true
+      volumeMounts:
+      - mountPath: "/mnt/data"
+        name: local
+  volumes:
+    - name: local
+      emptyDir: {}
+----
+<1> Install your NFS server image.
+
+..  Configure a `Service` resource object with the `nfs-server` role. For example, `nfs_service.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: nfs-service
+  labels:
+    role: nfs-server
+spec:
+  ports:
+  - name: 2049-tcp
+    port: 2049
+    protocol: TCP
+    targetPort: 2049
+  selector:
+    role: nfs-server
+  sessionAffinity: None
+  type: ClusterIP
+----
 
-<2> Set `storage.hive.sharedPVC.createPVC` to `true` and set the `storage.hive.sharedPVC.storageClass` to the name of a StorageClass with ReadWriteMany access mode. This will use dynamic volume provisioning to have a volume created automatically.
+..  Configure a `StorageClass` resource object. For example, `nfs_storageclass.yaml`:
++
+[source,yaml]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: nfs-server <1>
+provisioner: example.com/nfs
+parameters:
+  archiveOnDelete: "false"
+reclaimPolicy: Delete
+volumeBindingMode: Immediate
+----
+<1> Must exactly match the `[kind: PersistentVolume].spec.storageClassName` field value.
+
+
+[WARNING]
+====
+Configuration of your NFS storage, and any relevant resource objects, will vary depending on the NFS server image that you use for Metering storage.
+====
```

Comment:
@timflannagan @pruan-rht I included this note at the end of the procedure, if you think it might help for users who get NFS going with a different server image than the one in the docs.